oscarlab/graphene
# [Pal/lib] Download and patch TOML-parser library tomlc99

## Description of the changes 

This library will be used to parse Graphene manifest files written in TOML syntax. We patch the library slightly to remove unsupported `toml_parse_file()` and `toml_rtod()`, as well as `errno()` handling.

Must be rebased after https://github.com/oscarlab/graphene/pull/1926 is merged.

## How to test this PR? 

All tests must pass. I added a dummy `toml_parse()` to check that Graphene downloads, builds and uses TOML correctly.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1928)
